### ISSUE TITLE
Newest version of py3status fails to install via pip.

### ISSUE THREAD

**Issue by lainiwa** [ultrabug/py3status#1351]
I am trying to install the latest version of py3status(==3.9).
```bash
# pip3 --version
pip 10.0.1 from /usr/local/lib/python3.6/dist-packages/pip (python 3.6)
# pip3 install py3status==3.9
Collecting py3status==3.9
  Downloading https://files.pythonhosted.org/packages/57/b0/a875b5c767fd9b697e4de3cd261461f2da997a56f1f51ca5185a865d9c73/py3status-3.9.tar.gz (273kB)
    100% |████████████████████████████████| 276kB 3.3MB/s 
Installing collected packages: py3status
  Running setup.py install for py3status ... error
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-oun6n7g6/py3status/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-k7zq1rfl/install-record.txt --single-version-externally-managed --compile:
...
<multiple lines here>
...
    warning: no files found matching '*.py' under directory 'modules'
    writing manifest file 'py3status.egg-info/SOURCES.txt'
    error: can't copy 'py3status/modules': doesn't exist or not a regular file
    
    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-oun6n7g6/py3status/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-k7zq1rfl/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-oun6n7g6/py3status/
```

However when installing py3status==3.8 it works just fine.

**Comment by lasers** [comment]
Whilst waiting for a solution, you could try this: [Setting up a dev env](https://py3status.readthedocs.io/en/latest/contributing.html#setting-up-a-development-environment)

```python
# first step: using https
git clone https://github.com/ultrabug/py3status.git

# second step: pick install or develop
python setup.py install 

# py3status will be in
/usr/local/bin/py3status
```

**Comment by ultrabug** [comment]
Sorry about that, tbh I have no clue what changed in setuptools or pip since the MANIFEST.in has not changed... (but maybe that was the issue)

I've released 3.10 and dropped 3.9 from pypi, I've tested install is ok

Thanks for reporting this @lainiwa !
